src/app/modules/app/pages/classrooms/components/classroom-form/classroom-form.component.ts

```diff
@@ -229,6 +241,29 @@ export class ClassroomFormComponent implements OnInit {
     return classroom;
   }
 
+  buildMeetingFromForm(data: any) {
```

Comment:
The process of obtaining schedules data is done in the get classroom from form method, and actually you are using this method on the submit event, so it already has this data... You can extract the schedule from the classroom in the on submit method